libcli.c

```diff
@@ -3142,3 +3253,32 @@ void cli_unregister_all_commands(struct cli_def *cli) {
 void cli_unregister_all_filters(struct cli_def *cli) {
   cli_unregister_tree(cli, cli->commands, CLI_FILTER_COMMAND);
 }
+
+/*
```

Comment:
Do these need to exist? A point release shouldn't change existing functionality or the public API, and the cli_int* functions were specifically not part of the public API. I think it's fine to remove these completely.

Reply:
I think so actually.  For example, in our code after cli_init() is called we want to delete all of the pre-defined commands but keep all of the filters.  These two commands allow the user to delete either all filters or all commands.  The work in 1.10.0 wound up putting both filters and commands into the same cli->commands array, so I needed the 'unregister_tree()' call to indicate which kind of command to remove.  Could make a case that cli_unregister_tree needs to be cli_int_unregister_tree, but that also now gets into doing a code scrub and ensuring that *all* internal routines look like cli_int+* or CLI_* for consistency.